### DISCUSSION TITLE
The editor verify message field exceeds the display range

### DISCUSSION THREAD

**Discussion by Rain-0x01-39** [ppy/osu#28751] [Q&A]
![osu_2024-07-06_19-52-08](https://github.com/ppy/osu/assets/83620631/45c358d6-7c8c-472e-acb0-6cada05d8a9d)

Interesting.

**Comment by cdwcgt** [comment]
Maybe a dynamic IssueTable height is needed rather than just making it higher
![image](https://github.com/ppy/osu/assets/34775378/ef5b85f3-d10d-4e20-98db-1d3954828d43)
